src/app/services/progressesApi.ts

```diff
@@ -13,7 +14,15 @@ export const progressesApi = api.injectEndpoints({
                 },
             }),
         }),
+        getLatestProgress: builder.query({
+            query: (payload) => `/progresses/?${payload}`,
+            transformResponse: (response: taskProgressResponse) => {
```

Comment:
transformResponse function for the getLatestProgress endpoint is incorrectly returning an object with an empty id property if there are no results in the response